runtime/common/src/paras_registrar.rs

```diff
@@ -1052,9 +1055,21 @@ mod tests {
 			assert!(!Parachains::is_parachain(para_2));
 			assert!(Parachains::is_parathread(para_2));
 
-			// Both paras initiate a swap
-			assert_ok!(Registrar::swap(para_origin(para_1), para_1, para_2,));
-			assert_ok!(Registrar::swap(para_origin(para_2), para_2, para_1,));
+            // Both paras initiate a swap
+            // Swap between parachain and parathread
+            assert_ok!(Registrar::swap(para_origin(para_1), para_1, para_2,));
+            assert_ok!(Registrar::swap(para_origin(para_2), para_2, para_1,));
+            assert_eq!(
+                System::events(),
+                vec![EventRecord {
+                    phase: Phase::Initialization,
+                    event: Event::Registrar(paras_registrar::Event::Swapped {
+                        para_id: para_2,
+                        other_id: para_1
+                    }),
+                    topics: vec![],
+                }]
+            );
```

Comment:
I could have written the test this way:
```rust
assert!(System::events().iter().any(|r| matches!(
        r.event,
        Event::Registrar(paras_registrar::Event::Swapped { .. })
    )));
```
But in this way I was not able to check for the properties inside the Swapped Event.